aptos-move/framework/aptos-experimental/sources/trading/order_book/bulk_order_book_types.move

```diff
@@ -134,7 +126,7 @@ module aptos_experimental::bulk_order_book_types {
     /// - `vector<u64>`: Cancelled bid sizes corresponding to cancelled prices
     /// - `vector<u64>`: Cancelled ask prices (levels that crossed the spread)
     /// - `vector<u64>`: Cancelled ask sizes corresponding to cancelled prices
-    public(friend) fun new_bulk_order<M: store + copy + drop>(
+    public(package) fun new_bulk_order<M: store + copy + drop>(
```

Comment:
inside aptos framework (i.e. when we move this to 0x1), using friend is better (and the convention), as framework is huge, and you want to control access to only few other modules. 
so let's revert change to public(package)